webui/react/src/pages/F_ExpList/glide-table/menu.tsx

```diff
@@ -53,7 +53,7 @@ export const TableActionMenu: React.FC<TableActionMenuProps> = ({
         top: y,
         width: menuWidth,
       }}>
-      <Menu items={items} />
+      <Menu items={items} selectable={false} />
```

Comment:
q: what does this do?

Reply:
This prevents menu items from appearing as selected when the menu opens again.